rosie_map_controllers/MapStoring Request
---
int8 Response
